randovania/games/samus_returns/logic_database/Area 4 Central Caves.txt

```diff
@@ -925,6 +925,8 @@ Extra - asset_id: collision_camera_016
                   # High Jump
                   High Jump Boots
                   Super Jump (Intermediate) or Unmorph Extend (Intermediate)
+              # Spider Ball along ceiling and SWJ to climb up: https://youtu.be/1JuscAlYNIY
+              Single-wall Wall Jump (Expert) and Use Spider Ball
```

Comment:
is this weird shooting thing you make in the video required? if yes, sounds like that should be reflected in the requirements here.

Reply:
Yes it is. I guess movement (😖) will suffice